Cart Flow/Custom Views/Buttons/CFButton.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CFButton.swift
+//  Cart Flow
+//
+//  Created by Scott Quintana on 8/6/20.
+//  Copyright © 2020 Scott Quintana. All rights reserved.
+//
+
+import UIKit
+
+class CFButton: UIButton {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    
+    convenience init(backgroundColor: UIColor, title: String) {
+        self.init(frame: .zero)
+        self.backgroundColor = backgroundColor
+        self.setTitle(title, for: .normal)
+    }
+    
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    
+    func configure() {
+        layer.cornerRadius      = 16
```

Comment:
Some weird spacing here between `layer.cornerRadius` and `= 16`. I won't comment any more in this review on code style, but some people are real sticklers for that in code review. Generally, developers on a codebase agree to a style guide and try to stick to it for the project. On solo projects, I like to pick a style guide and run with it (you seem to have done a pretty good job on this project keeping things consistent with your style), and on projects with others I try to fit in with their style or follow their style guide. There's a ton of suggested style guides to abide by if you search the internet. Here's on as an example: https://github.com/raywenderlich/swift-style-guide